### DISCUSSION TITLE
How to disable lsp event indicate?

### DISCUSSION THREAD

**Discussion by MedievalSteel** [joaotavora/eglot#1160] [Q&A]
![Screenshot 2023-02-01 150033](https://user-images.githubusercontent.com/74715689/215973805-d8b4ab70-5aa2-4d2e-adbc-779341b0a8c0.jpg)

It blink frequently.

**Comment by MedievalSteel** [comment]
have any config?

**Reply by joaotavora** [reply]
not have.  you make?
